coverage/.resultset.json

```diff
@@ -1,6 +1,33 @@
 {
```

Comment:
Hahahaha I don't know what's happening here, but I think it'll be solved when we get everything merged and Steffany does whatever it was that she was going to do.